src/styles/less/GamemodeStyles.less

```diff
@@ -1,8 +1,19 @@
 //** Gameplay Styles
 #gameplay {
+  @media (max-width: 800px) {
+    resize: both !important;
+    overflow: hidden !important;
+    max-width: auto !important;
```

Comment:
Is it necessary to use !important here? I've noticed that it's used quite a bit in the CSS for this project, but it's considered bad practice and should be avoided.

Reply:
Ok this is great feedback. I will look into this.